lib/matplotlib/backends/backend_pdf.py

```diff
@@ -250,6 +250,57 @@ def _datetime_to_pdf(d):
     return r
 
 
+def _calculate_quad_point_coordinates(x, y, width, height, angle=0):
+    """
+    Uses matrix maths to calculate the coordinates of a
+    rectangle when rotated by angle around x, y
+    """
+
+    angle = math.radians(angle)
```

Comment:
It would be slightly more efficient to just compute cos and sin once, but since this is not that heavily used it can wait.